docs/analyzer/user_guide.md

```diff
@@ -1745,6 +1753,348 @@ then the results of the analysis can be printed with
+          }
+        },
+        {
+          "file": {
+            "id": "/home/username/dummy/simple/main.cpp",
+            "path": "projects/dummy/main.cpp",
+            "original_path": "/home/username/projects/dummy/main.cpp"
+          },
+          "line": 3,
+          "column": 1,
+          "message": "Entered call from 'main'",
+          "range": {
+            "start_line": 3,
+            "start_col": 1,
+            "end_line": 3,
+            "end_col": 1
+          }
+        },
+        {
+          "file": {
+            "id": "/home/username/dummy/simple/main.cpp",
+            "path": "projects/dummy/main.cpp",
+            "original_path": "/home/username/projects/dummy/main.cpp"
+          },
+          "line": 5,
+          "column": 12,
+          "message": "Division by zero",
+          "range": {
+            "start_line": 5,
+            "start_col": 12,
+            "end_line": 5,
+            "end_col": 12
+          }
+        }
+      ],
+      "bug_path_positions": [
+        {
+          "range": {
+            "start_line": 9,
+            "start_col": 3,
+            "end_line": 9,
+            "end_col": 8
+          },
+          "file": {
+            "id": "/home/username/dummy/simple/main.cpp",
+            "path": "projects/dummy/main.cpp",
+            "original_path": "/home/username/projects/dummy/main.cpp"
+          }
+        },
+        {
+          "range": {
+            "start_line": 9,
+            "start_col": 14,
+            "end_line": 9,
+            "end_col": 14
+          },
+          "file": {
+            "id": "/home/username/dummy/simple/main.cpp",
+            "path": "projects/dummy/main.cpp",
+            "original_path": "/home/username/projects/dummy/main.cpp"
+          }
+        }
+      ],
+      "notes": [],
+      "macro_expansions": [
+        {
+          "name": "DIV",
+          "file": {
+            "id": "/home/username/dummy/simple/main.cpp",
+            "path": "projects/dummy/main.cpp",
+            "original_path": "/home/username/projects/dummy/main.cpp"
+          },
+          "line": 5,
+          "column": 10,
+          "message": "1 / p",
+          "range": {
+            "start_line": 5,
+            "start_col": 10,
+            "end_line": 5,
+            "end_col": 10
+          }
+        }
+      ]
+    }
+  ]
+}
+```
+
+- `version` (number): version number. If the format of the JSON output will
+  change this value will be incremented too. Currently supported values: `1`.
+- `reports` (list): list of [Report objects](#report-object).
+
+#### Report object
+- `file` (File): file where the report was found in. For more information
+  [see](#file-object).
+- `line` (number): line number.
+- `column` (number): column number.
+- `message` (str): message reported by the checker.
+- `checker_name` (str): identifier of the rule (checker) that was evaluated
+  to produce the result.
+- `severity` (str | null): CodeChecker severity level (optional).
```

Comment:
Can you list in the docs, what are the possible values of severity? It's useful in order to filter severities > LOW for example. That was trivial when severity was numeric.

Reply:
Done: #3604

Reply:
Awesome, thanks!